const trycatch = (try_fn, catch_fn) => {
    try {
        try_fn()
    } catch (e) {
    println(e)
       catch_fn()
    }
}

const add = (a, b) => {
    return trycatch(() => {
        const g = 120
        g * a + b
    }, () => 512)
}

add(5, "1").println()

// try {
//     println(x * 100 / "j")
// } catch (e) {
//    println(5)
// }